index.html

```diff
@@ -155,32 +160,32 @@ <h2 class="title">Tonic</h2>
         <!--End card section 1-->
 
         <!--card section 1-->
-        <div class="card card1">
```

Comment:
The work section has broken images for all the projects on both mobile and desktop views. The title should have `#091E42` color. Moreover please use the hover effect for the `See Project` button.
![picture](https://user-images.githubusercontent.com/35504149/172493176-71626a13-e6be-4495-ac64-d93cd7bd5d47.png)